test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java

```diff
@@ -77,8 +77,11 @@ public void testFailure(Failure failure) throws Exception {
         }
         b.append(failure.getDescription().getClassName());
 
-        final String methodName = failure.getDescription().getMethodName();
+        String methodName = failure.getDescription().getMethodName();
         if (methodName != null) {
+            // Method names only have spaces if used with -Dtests.iters where the seed is appended.
+            // Here we strip out anything from the first space, leaving the real method name intact.
+            methodName = methodName.split(" ")[0];
```

Comment:
I don't think this is the case. For things like YAML tests the names very often have spaces. This would then cause those repro lines to be corrupted.

Reply:
You're right, I had not considered parameterized tests. In fact, the seed is put into the same curly-braced section as parameters.